.github/workflows/release.yml

```diff
@@ -39,12 +25,12 @@ jobs:
         HostName github.com
         IdentityFile ~/.ssh/id_rsa
         EOT
-    - name: Release to GitHub Pages
+    - name: Release Documentation to GitHub Pages
       env:
         USE_SSH: true
-        GIT_USER: git
+        GIT_USER: irrelevant
       run: |
-        git config --global user.email "actions@gihub.com"
-        git config --global user.name "gh-actions"
+        git config --global user.email "irrelevant@completely.irrelevant"
+        git config --global user.name "irrelevant"
```

Comment:
If the username/email are not relevant, is it possible to remove some of them?

Reply:
@jjtang1985, sadly both are needed to preconfigure git so that it doesn't complain and fails with non-zero exit code. A similar approach is used in Docusaurus docs for Circle CI. I don't know a more beautiful workaround for it.